packages/jsii-pacmak/lib/targets/go/util.ts

```diff
@@ -46,8 +46,39 @@ export function getFieldDependencies(fields: GoTypeMember[]): Package[] {
   }, []);
 }
 
+// export function getParameterDependencies(params: GoParameter[]): Package[] {
+//   return params.reduce((accum: Package[], param) => {
+//     return param.reference?.type?.pkg
+//       ? [...accum, param.reference?.type.pkg]
+//       : accum;
+//   }, []);
+// }
```

Comment:
If this was needed, it would not be commented... And if it's not needed, it can be removed :)

Reply:
Whoops, sloppiness on my part. Will remove.